cmd/policygen/main.go

```diff
@@ -28,7 +28,7 @@ import (
 
 var (
 	configPath = flag.String("config_path", "", "Path to the Policy Generator config.")
-	statePath  = flag.String("state_path", "", "Path to Terraform state in json format.")
+	statePath  = flag.String("state_path", "", "Path to Terraform states in Terraform native .tfstate format. If a directory path is given, then all .tfstate files will be read recursively.")
```

Comment:
Does it read recursively in GCS buckets? That was actually my main FR

Reply:
Not yet. It's reading from a local dir first.